Traceback (most recent call last):
  File "/home/corrin/miniconda3/envs/book/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/corrin/miniconda3/envs/book/lib/python3.8/site-packages/nbclient/client.py", line 1093, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/corrin/miniconda3/envs/book/lib/python3.8/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/corrin/miniconda3/envs/book/lib/python3.8/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/home/corrin/miniconda3/envs/book/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/home/corrin/miniconda3/envs/book/lib/python3.8/site-packages/nbclient/client.py", line 559, in async_execute
    await self.async_execute_cell(
  File "/home/corrin/miniconda3/envs/book/lib/python3.8/site-packages/nbclient/client.py", line 854, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/home/corrin/miniconda3/envs/book/lib/python3.8/site-packages/nbclient/client.py", line 756, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# using rename()
ChickWeight <- rename(ChickWeight, Weight = weight)
------------------

Error: Can't rename columns that don't exist.
[31mâœ–[39m Column `weight` doesn't exist.
Traceback:

1. rename(ChickWeight, Weight = weight)
2. rename.data.frame(ChickWeight, Weight = weight)
3. tidyselect::eval_rename(expr(c(...)), .data)
4. rename_impl(data, names(data), as_quosure(expr, env), strict = strict, 
 .     name_spec = name_spec)
5. eval_select_impl(x, names, {
 .     {
 .         sel
 .     }
 . }, strict = strict, name_spec = name_spec, type = "rename")
6. with_subscript_errors(vars_select_eval(vars, expr, strict, data = x, 
 .     name_spec = name_spec, uniquely_named = uniquely_named, allow_rename = allow_rename, 
 .     type = type), type = type)
7. tryCatch(instrument_base_errors(expr), vctrs_error_subscript = function(cnd) {
 .     cnd$subscript_action <- subscript_action(type)
 .     cnd$subscript_elt <- "column"
 .     cnd_signal(cnd)
 . })
8. tryCatchList(expr, classes, parentenv, handlers)
9. tryCatchOne(expr, names, parentenv, handlers[[1L]])
10. value[[3L]](cond)
11. cnd_signal(cnd)
12. rlang:::signal_abort(x)
ERROR: Error: Can't rename columns that don't exist.
[31mâœ–[39m Column `weight` doesn't exist.


